set.mm

```diff
@@ -5467,6 +5467,14 @@ use disjunction (although this is not required since definitions are
       ( wa simpl syl2anc ) ABGACDABHEFI $.
   $}
 
+  ${
+    sylibda.1 $e |- ( ph -> ( ps <-> ch ) ) $.
+    sylibda.2 $e |- ( ( ph /\ ch ) -> th ) $.
+    $( A syllogism deduction.  (Contributed by SN, 16-Jul-2024.) $)
+    sylibda $p |- ( ( ph /\ ps ) -> th ) $=
+      ( biimpa syldan ) ABCDABCEGFH $.
+  $}
+
```

Comment:
uh oops, I just realized this probably should've been named `sylbida` (swaps i and b)

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4220/commits/2ee397d50a6ce88f8591da158032b3815626e6c9